staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_utils_test.go

```diff
@@ -299,6 +299,73 @@ func assertInternalLbResourcesDeleted(t *testing.T, gce *Cloud, apiService *v1.S
 	assert.Nil(t, ip)
 }
 
+func assertInternalLBAnnotation(t *testing.T, gce *Cloud, clusterID string, svc *v1.Service, protocol v1.Protocol) {
```

Comment:
assertILBAnnotationSet? and the other function assertILBAnnotationUnset?

Reply:
Thanks! I like those function names better too.